src/pages/settings/Profile/CustomStatus/StatusPage.js

```diff
@@ -37,19 +37,19 @@ function StatusPage({draftStatus, currentUserPersonalDetails}) {
     const customStatus = draftEmojiCode ? `${draftEmojiCode} ${draftText}` : `${currentUserEmojiCode || ''} ${currentUserStatusText || ''}`;
     const hasDraftStatus = !!draftEmojiCode || !!draftText;
 
+    const clearStatus = () => {
+        User.clearCustomStatus();
+        User.clearDraftCustomStatus();
+    };
+
+    const onNavigateBackToSettingsPage = useCallback(() => Navigation.goBack(ROUTES.SETTINGS_PROFILE, false, true), []);
```

Comment:
based on Reviewer Checklist we should avoid naming event handlers like `onSomething` so please change this handler's name to something like `navigateBackToSettingsPage`

Reply:
👍🏻